### PR TITLE
Update Zen extension logic to capture extensions before backup, standardize some values across files

### PR DESCRIPTION
**Pull request by bluzarraga**
See issues 60195, 61131, and 61134

### CODE REVIEW

File: velero/schedule/zen5-br-scripts-cm.yaml [hunk 1]
```diff
@@ -355,12 +353,35 @@ data:
...
+                        extensions="$ext_name,"
+                    else
+                        extensions="$extensions,$ext_name"
+                    fi
+                done
+                
+                extensions=$(echo $extensions | tr "," " ")
+                echo $extensions
+                
+                cat <<< $extensions > /tmp/extensions.txt
```
Comment (joshua-bj) [inline velero/schedule/zen5-br-scripts-cm.yaml:373]: @bluzarraga , Hi Ben, the `/tmp/` folder is not in the backup scope, I guess you need persist this file at `/zen5/zen-backup` or its sub folder

Comment (joshua-bj) [reply velero/schedule/zen5-br-scripts-cm.yaml:373]: Oh, I see there is a new "emptyDir" mounted at the backup pod, but I'm wondering why you do not use the `/zen5/zen-backup`. The emptyDir is usually for temporary persistency request like cache. Seems like it's any pattern to be considered at the backup. Just my gut feeling.

Comment (bluzarraga) [reply velero/schedule/zen5-br-scripts-cm.yaml]: changed the directory to /zen5/extensions so it is in the same parent directory as the rest of the backup

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**